melos.yaml

```diff
@@ -1,64 +1,73 @@
 name: firmware_updater_workspace
+sdkPath: .fvm/flutter_sdk
 
 packages:
-  - packages/**
+  - packages/*
```

Comment:
what's the story with changing the glob here? Was this not working previously or is it a change on the melos side that we're following?